src/components/Header.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import Header from './Header'
+
+test('renders test text', () => {
+	render(<Header />)
+	const linkElement = screen.getByText(/test/i)
+	expect(linkElement).toBeInTheDocument()
```

Comment:
It would be nice to also replace old variables naming. :wink:
```suggestion
	const headerElement = screen.getByText(/test/i)
	expect(headerElement).toBeInTheDocument()
```
Maybe even not just retrieve and validate a general text, but rather exact element, like in this case `<h1>`. :thinking:

Reply:
There might not be an `<h1>` element or even any text after we design the Header component as it's supposed to look. **I put there a test now for the sole purpose of Header component being used as an example of a component** that's supposed to show all structure and all standards we're supposed to do when creating further components. 

Therefore I've only commited your suggestion fixing incorrect variable naming and I will write a different, possibly more complex test when we know what we want to test in this component and how to do so.